frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx

```diff
@@ -139,12 +159,22 @@ export default function ApplyFlowTypeOfApplication() {
             options={[
               {
                 value: ApplicantType.Personal,
-                children: t('apply:eligibility.type-of-application.radio-options.personal'),
+                children: <Trans ns={handle.i18nNamespaces} i18nKey="eligibility.type-of-application.radio-options.personal" />,
```

Comment:
Add namespace key prefix `apply:` to `i18nKey` prop. Do the same to others.
```suggestion
                children: <Trans ns={handle.i18nNamespaces} i18nKey="apply:eligibility.type-of-application.radio-options.personal" />,
```